conf/DEV.public.conf

```diff
@@ -30,4 +30,10 @@ settingsSource {
       path="~/.gu/support-admin-console/switches.json"
     }
   }
+  amounts {
+    s3 {
```

Comment:
could be amounts.s3 the same as the others?

Reply:
personally I'm not a fan of putting so much in config, the only thing that changes is the DEV/PROD part so all the rest could be hard coded.  In my mind config is useful for changing things without a recompile, and we would generally rebuild every time we edit these files anyway so we are just losing type safety and IDE support for nothing..

Reply:
good point, will have a think

Reply:
I'll revisit this in a follow up PR as this one is big enough already!

Reply:
that's a sensible approach!